ResourceManager/owlbot.py

```diff
@@ -57,16 +52,6 @@
     r"\$transportConfig, and any \$serviceAddress",
     r"$transportConfig, and any `$apiEndpoint`")
 
-# fix year
-s.replace(
-    '**/*Client.php',
-    r'Copyright \d{4}',
-    'Copyright 2021')
-s.replace(
-    'tests/**/*Test.php',
-    r'Copyright \d{4}',
-    'Copyright 2021')
-
 # Change the wording for the deprecation warning.
 s.replace(
     'src/*/*_*.php',
```

Comment:
`ResourceManager` falls in the category of APIs which are not yet GA. We can add excludes for `*_*.php` and `class_alias`, and remove the block to replace the wording for the deprecation warning in `*_*.php`.

Reply:
Done